.github/workflows/ci.yaml

```diff
@@ -1,9 +1,9 @@
 name: ci
 on:
   push:
-    branches: [main]
+    branches: [v2] # TODO: Switch to main once v2 is merged
   pull_request:
-    branches: [main]
+    branches: [v2] # TODO: Switch to main once v2 is merged
```

Comment:
FWIW, I'm not sure this will actually take effect until it is merged to main.

So it might be a reasonable idea to propagate some of this config to a normal PR agains main branch, too. In that case, it should be `[main, v2]`.